### PR TITLE
update navbar container

### PR DESCRIPTION
**Pull request by bravogeorge**
https://github.com/react-bootstrap/react-bootstrap/issues/5752

Updated navbar examples in https://react-bootstrap-v5.netlify.app/components/navbar/#navbars-containers to add the .container tag per @kyletsang notes.

Preview: https://deploy-preview-5766--react-bootstrap-v5.netlify.app/components/navbar/

### CODE REVIEW

**Review by kyletsang — CHANGES_REQUESTED** [review]
Looks good!

A couple more examples need the container then we're good to merge:
https://deploy-preview-5766--react-bootstrap-v5.netlify.app/components/navbar/#navbars-colors
https://deploy-preview-5766--react-bootstrap-v5.netlify.app/components/navbar/#navbars-mobile-friendly

**Comment by bravogeorge** [comment]
> Looks good!
> 
> 
> 
> A couple more examples need the container then we're good to merge:
> 
> https://deploy-preview-5766--react-bootstrap-v5.netlify.app/components/navbar/#navbars-colors
> 
> https://deploy-preview-5766--react-bootstrap-v5.netlify.app/components/navbar/#navbars-mobile-friendly

Ah, okay. I didn't see the container tag on the getbootstrap page so I thought the reactbootstrap also didn't require it. I will make those updates as soon as I get
Off work! Thank you again for all your help!

**Comment by bravogeorge** [comment]
@kyletsang I added the remaining navbar examples. Hopefully they all check out!

File: www/src/examples/Navbar/Collapsible.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+  <Container>
   <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
   <Navbar.Toggle aria-controls="responsive-navbar-nav" />
   <Navbar.Collapse id="responsive-navbar-nav">
+    <Container>
```
Comment (kyletsang) [inline www/src/examples/Navbar/Collapsible.js:6]: The `Container` under `Navbar.Collapse` isn't needed.

Comment (bravogeorge) [reply www/src/examples/Navbar/Collapsible.js]: Ack, got it. Removed container tag inside navbar collapse from collapsible.js. Should be good to go. Thank you so much for all your help @kyletsang

**Review by kyletsang — APPROVED** [review]

**Comment by kyletsang** [comment]
Thanks!

**Pull request merged**